### DISCUSSION TITLE
Is there a way to reset the date picker to the current calendar page?

### DISCUSSION THREAD

**Discussion by davestewart** [unovue/reka-ui#1479] [Q&A] [answered]
Some calendars have a "Today" or "Reset" button for when you have navigated to a new calendar page.

This should not pick a date, but just reset the page to the starting view.

Is this functionality available?

**Comment by zernonia** [comment, marked as answer]
Yes it's available. Check this out. It also showcase the way to control the open/close state of calendar you asked [here](https://github.com/unovue/radix-vue/discussions/1480)

https://stackblitz.com/edit/ik17p7?file=src%2FApp.vue

**Reply by davestewart** [reply]
Thanks! I appreciate the input

**Reply by davestewart** [reply]
Also, would suggest adding this to the docs / demo code
